providers/vcd/resources/vcd.lr

```diff
@@ -120,6 +120,9 @@ vcd.networkPool {
 
 // VMware Cloud Director External Network
 private vcd.externalNetwork @defaults("name") {
+  // URN of the network.
+  // Deprecated: Please use the urn field instead.
```

Comment:
Can we shove this on one line. Otherwise the docs get generated with a funky comma after the period